routes/movies.js

```diff
@@ -0,0 +1,44 @@
+const Movie = require('../models/movie')
+const Celebrity = require('../models/celebrity')
+
+const router = require("express").Router(); // What does that mean ?
+
+router.get('/', (req,res,next) => {
+    Movie.find()
+    .then(moviesList => {
+        console.log(moviesList)
+        res.render('movies/index', {moviesList})
+    })
+    
+})
+
+router.get('/new', (req,res,next) => {
+    Celebrity.find()
+    .then(celebrityList =>{
+        const celebritiesNames = celebrityList.map(function(name){
+            return name.name
+        })
+        console.log(celebrityList)
+        res.render('movies/new', {celebritiesNames})
```

Comment:
You would need to pass the celebrity IDs to the hbs file. So better pass the whole `celebrityList` array to the hbs and then filtering the names and ids there. The reason you need the IDs in the hbs, is that when the user select a celebrity, you need the value of the selection to be the celebrity ID, since this is what you are going to save in the database inside the cast array.